cmd/nri-ecs.go

```diff
@@ -34,91 +34,59 @@ func main() {
 		os.Exit(1)
 	}
 
-	if args.DebugMode {
-		v3.DebugECSEndpoint()
-		os.Exit(0)
-	}
-
 	httpClient := ecs.ClientWithTimeout(5 * time.Second)
 
-	taskMetadataEnpoint, found := v3.TaskMetadataEndpoint()
+	taskMetadataEnpoint, found := metadata.TaskMetadataEndpoint()
 	if !found {
-		ecsIntegration.Logger().Errorf("unable to find task metadata endpoint")
+		log.Error("unable to find task metadata endpoint")
 		os.Exit(1)
 	}
 
-	body, err := v3.MetadataResponse(httpClient, taskMetadataEnpoint)
+	body, err := metadata.MetadataResponse(httpClient, taskMetadataEnpoint)
 	if err != nil {
-		ecsIntegration.Logger().Errorf(
+		log.Error(
 			"unable to get response from v3 task metadata endpoint (%s): %v",
 			taskMetadataEnpoint,
 			err,
 		)
 		os.Exit(1)
 	}
 
-	var taskMetadata v3.TaskResponse
+	if args.DebugMode {
```

Comment:
shouldn't we set the debug level if args.verbose and change this to log.Debug?

Reply:
yes that make sense!